content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx

```diff
@@ -202,7 +203,9 @@ const Stop: FC<{ stop: Stop; isFirstStop: boolean }> = ({
               }}
               v={{ size: 'l', properties: ['margin-bottom'] }}
             >
-              <StandaloneTitle>{standaloneTitle}</StandaloneTitle>
+              <StandaloneTitle id={dasherize(`${standaloneTitle}`)}>
```

Comment:
nit: I think you can do away with the interpolation here

```suggestion
              <StandaloneTitle id={dasherize(standaloneTitle)}>
```

Reply:
Ah i remember why i did this now, standaloneTItle is a `prismicT.RichTextField` so I'm ensuring I change it to a `string` to keep Typescript and the dasherize function happy. The other places I've placed dasherize work with `title` which is a `string`.